test/unit/CMakeLists.txt

```diff
@@ -119,6 +119,15 @@ PDAL_ADD_TEST(pdal_io_ept_reader_test
     INCLUDES
         ${PDAL_JSONCPP_INCLUDE_DIR}
 )
+PDAL_ADD_TEST(pdal_io_ept_addon_writer_test
+    FILES
+        io/EptAddonWriterTest.cpp
+    LINK_WITH
+        ${PDAL_JSONCPP_LIB_NAME}
+        ${GDAL_LIBRARY}
```

Comment:
Why do you need GDAL?  Are you calling GDAL functions from the test?

Reply:
It's not needed - this was a copy-paste from the Reader test, which uses the CropFilter thus linked GDAL.  Removed at fce635d.